tests/mocks/repository/lib/mock_kvset.c

```diff
@@ -494,6 +465,36 @@ _kvset_cursor_next(struct element_source *es, void **element)
     return true;
 }
 
+merr_t
+_kvset_iter_create(
+    struct kvset *           kvset,
+    struct workqueue_struct *workq,
+    struct workqueue_struct *vra_wq,
+    struct perfc_set *       pc,
+    enum kvset_iter_flags    flags,
+    struct kv_iterator **    handle)
+{
+    struct mock_kv_iterator *iter;
+
+    iter = calloc(1, sizeof(*iter));
+    if (!iter)
+        return merr(ENOMEM);
+
+    memset(iter->tripwire, 0xaa, sizeof(iter->tripwire));
+    iter->kvset = (struct mock_kvset *)kvset;
+    iter->src = 0;
+    iter->nextkey = 0;
+
+    iter->kvi.kvi_ops = &mock_kvset_ops;
+    iter->kvi.kvi_context = iter;
+    iter->kvi.kvi_eof = false;
+    iter->kvi.kvi_es = es_make(_kvset_cursor_next, NULL, NULL);
+
+    *handle = &iter->kvi;
+
+    return 0;
+}
+
```

Comment:
We should stop using leading underscores for these mocks and use function_mtf() or whatever.  Some day...

Reply:
Yea has to do with the macro prepending the "section" or whatever you want to call it. Otherwise I would name it differently. Kinda just stuck like this at the moment.

Reply:
We can update the MOCK_SET macro to use a different prefix, and then update all the code that uses the macro.